core/spoofax.compiler/src/main/java/mb/spoofax/compiler/adapter/TegoRuntimeAdapterCompiler.java

```diff
@@ -0,0 +1,73 @@
+package mb.spoofax.compiler.adapter;
+
+import mb.common.util.ListView;
+import mb.pie.api.ExecContext;
+import mb.pie.api.None;
+import mb.resource.hierarchical.ResourcePath;
+import mb.spoofax.compiler.language.ClassLoaderResourcesCompiler;
+import mb.spoofax.compiler.language.TegoRuntimeLanguageCompiler;
+import mb.spoofax.compiler.util.ClassKind;
+import mb.spoofax.compiler.util.GradleConfiguredDependency;
+import mb.spoofax.compiler.util.Shared;
+import mb.spoofax.compiler.util.TemplateCompiler;
+import org.immutables.value.Value;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.io.Serializable;
+
+/**
+ * Tego runtime adapter compiler.
+ */
+@Value.Enclosing
+public class TegoRuntimeAdapterCompiler {
```

Comment:
This class doesn't seem to do anything. Do you see a future need for it? If not, it can be removed.

Reply:
I removed `TegoRuntimeLanguageCompiler` instead and moved the dependency to the `TegoRuntimeAdapterCompiler`.